crates/sui-cost/tests/snapshots/snapshot_tests__good_snapshot.snap

```diff
@@ -15,15 +15,15 @@ MergeCoin:
   storageCost: 32
   storageRebate: 0
 ? SplitCoin: 1
-: computationCost: 527
+: computationCost: 514
```

Comment:
Is the drop from natives basically being zeroed out for now?

Reply:
That was my guess. Plus so much has changed that I'm not 100% sure what else is causing this. But I didn't sweat it since we're reworking this anyways with fresh logic.